### ISSUE TITLE
[a11y] Cannot switch login screen language

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1601]
**Describe the bug**
I cannot switch the language on the login screen.

![chrome_frcNUHZhFq](https://user-images.githubusercontent.com/37256050/168604220-58c32c8a-20e2-440a-a3c1-ae2c1bb3a450.gif)

db: `nm3`
